apps/web/src/components/marketing/plans.tsx

```diff
@@ -1,96 +1,44 @@
 import Link from "next/link";
 import { Check } from "lucide-react";
 
+import type { PlanProps } from "@/config/plans";
+import { plansConfig } from "@/config/plans";
 import { cn } from "@/lib/utils";
 import { Shell } from "../dashboard/shell";
+import { LoadingAnimation } from "../loading-animation";
 import { Button } from "../ui/button";
 
-interface Plan {
-  title: string;
-  description: string;
-  cost: number | string;
-  features: string[];
-  action?: {
-    text: string;
-    link: string;
-  };
-  disabled?: boolean;
-}
-
-const plans: Record<"hobby" | "pro" | "enterprise", Plan> = {
-  hobby: {
-    title: "Hobby",
-    description: "Get started now and upgrade once reaching the limits.",
-    cost: 0,
-    features: [
-      "5 monitors",
-      "1 status page",
-      "subdomain",
-      "10m, 30m, 1h checks",
-    ],
-    action: {
-      text: "Start Now",
-      link: "/app/sign-up?plan=hobby",
-    },
-  },
-  pro: {
-    title: "Pro",
-    description: "Scale and build monitors for all your services.",
-    cost: 29,
-    features: [
-      "20 monitors",
-      "5 status page",
-      "custom domain",
-      "1m, 5m, 10m, 30m, 1h checks",
-      "5 team members",
-    ],
-    action: {
-      text: "Start Now",
-      link: "/app/sign-up?plan=pro",
-    },
-  },
-  enterprise: {
-    title: "Enterprise",
-    description: "Dedicated support and needs for your company.",
-    cost: "Lets talk",
-    features: [],
-    action: {
-      text: "Schedule call",
-      link: "https://cal.com/thibault-openstatus/30min",
-    },
-  },
-};
-
 export function Plans() {
   return (
     <Shell>
       <div className="grid gap-4 md:grid-cols-2 md:gap-0">
         <Plan
-          {...plans.hobby}
+          {...plansConfig.free}
           className="md:border-border/50 md:border-r md:pr-4"
         />
-        <Plan {...plans.pro} className="md:pl-4" />
+        <Plan {...plansConfig.pro} className="md:pl-4" />
         <Plan
-          {...plans.enterprise}
+          {...plansConfig.enterprise}
           className="md:border-border/50 col-span-full md:mt-4 md:border-t md:pt-4"
         />
       </div>
     </Shell>
   );
 }
 
-interface Props extends Plan {
+interface Props extends PlanProps {
   className?: string;
 }
 
-function Plan({
+export function Plan({
```

Comment:
We are using it also in setting maybe extract it to components shared plan

Reply:
We now use only one component for both, marketing and settings. Will extract it later!